data/tasks.txt

```diff
@@ -4,9 +4,6 @@ T|0|test 09/09/09
 T|1|test1 16/10/2019 
 T|0|read books
 T|0|sleep
-<<<<<<< HEAD
 T|0|sleep
-milk|2|null
-=======
->>>>>>> upstream/master
+T|2|null
```

Comment:
Do we still need tasks.txt file? Can delete?

Reply:
I wanted to get rid of all tasks related files, but am afraid of any implications.
Go ahead and do whatever you want with it, but please make sure there's no compile error.